README.md

```diff
@@ -81,4 +82,59 @@ look like this:
     |-- <QBIC sample code>.fastq.gz.sha256sum
 ```
 
+### NGS single-end / paired-end data with metadata (deprecated)
 
+**Disclaimer!**  
+This data format is targeted for a single use case and should not be
+used for general data registration purposes. Please use the
+[NGS single-end / paired-end data](#ngs-single-end--paired-end-data)
+format for now.
+
+**Responsible dropbox:**
+[QBiC-register-imgag-dropbox](drop-boxes/register-imgag-dropbox)
+
+**Resulting data model in openBIS**  
+Q_TEST_SAMPLE -> Q_NGS_SINGLE_SAMPLE_RUN (with sample code) -> DataSet
+of type Q_NGS_RAW_DATA (directory with raw sequencing files contained)
```

Comment:
I cannot verify if this is the structure. I did not find an openbis project for that so I will trust you here :)